pkg/status/status.go

```diff
@@ -0,0 +1,23 @@
+package status
+
+import "strings"
+
+type Stage string
+type State string
+
+const (
+	// Stages of build process
+	CONFIGURE   Stage = "configure"
+	APK_INSTALL Stage = "apk-install"
+	BUILD       Stage = "build"
+
+	// States of build's stages
+	FAILED     State = "FAILED"
+	INPROGRESS State = "INPROGRESS"
+	SUCCESS    State = "SUCCESS"
```

Comment:
it's fine that you have used `SUCCESS` here. Just make sure to communicate to the frontend the same